.travis.yml

```diff
@@ -8,3 +8,11 @@ before_script:
 
 script:
   - bundle exec rspec
+
+psql:
```

Comment:
Don't forget to add instructions for the test version of the database. Have you thought about adding .sql files to allow the user to run the queries instead of typing them?